api/test/integration/env/tools/healthcheck_utils.py

```diff
@@ -44,11 +47,20 @@ def get_response(url, headers):
     Dict
         API response for the request.
     """
+    import enum
+    import requests
     import urllib3
+
+    class RequestMethodWrapper(enum.Enum):
+        GET = requests.get
+        POST = requests.post
+        PUT = requests.put
+        DELETE = requests.delete
+
     # Disable insecure https warnings (for self-signed SSL certificates)
     urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
-    from requests import get
-    request_result = get(url, headers=headers, verify=False)
+
+    request_result = getattr(RequestMethodWrapper, request_method)(url, headers=headers, verify=False)
```

Comment:
That is not how you use an `Enum`. If you want to use `getattr` (which I believe it is the best option), you can directly use `getattr(requests, request_method)(*args, **kwargs)`. Consider changing to `request_method.lower()` if needed.

Reply:
`RequestMethodWrapper` removed in favour of using `getattr(requests, request_method)(*args, **kwargs)`.